### PR TITLE
update to WindowsAzure.Storage 7

### PR DESCRIPTION
**Pull request by SimonCropp**
also remove inferred nuget dependencies

### CODE REVIEW

File: packaging/nuget/nservicebus.azurestoragequeues.nuspec [hunk 1]
```diff
@@ -12,14 +12,10 @@
     <requireLicenseAcceptance>$requireLicenseAcceptance$</requireLicenseAcceptance>
     <description>NServiceBus Transport for the Windows Azure Storage Queues</description>
     <copyright>$copyright$</copyright>
-    <tags>nservicebus servicebus msmq cqrs publish subscribe</tags>
+    <tags>$tags$</tags>
     <dependencies>
-      <dependency id="WindowsAzure.Storage" version="[6.0.0, 7.0.0)" />
-      <dependency id="Microsoft.Data.Edm" version="[5.6.0, 6.0.0)" />
-      <dependency id="Microsoft.Data.OData" version="[5.6.4, 6.0.0)" />
+      <dependency id="WindowsAzure.Storage" version="[7.0.0, 6.0.0)" />
```
Comment (SeanFeldman) [inline packaging/nuget/nservicebus.azurestoragequeues.nuspec]: @SimonCropp shouldn't it be `[7.0.0, 8.0.0)`?

Comment (SimonCropp) [inline packaging/nuget/nservicebus.azurestoragequeues.nuspec]: doh. fixed

**Comment by SeanFeldman** [comment]
As @danielmarbach would say, Das ist fantastisch!

**Comment by SeanFeldman** [comment]
To avoid tweets like this: https://twitter.com/dgcaron/status/719525246227914752

**Comment by danielmarbach** [comment]
Could we backport this to 6.3 or something?

**Pull request merged**